src/widget/city_without_overlay.c

```diff
@@ -373,27 +432,30 @@ static int get_warehouse_flag_image_id(const building *b)
     return image_id;
 }
 
-static void draw_warehouse_flag(const building *b, int x, int y, color_t color_mask)
```

Comment:
Why remove the `const`?

Reply:
Because previously I change some building property inside thus methods but I remove the code and now we can undo this change.